contracts/governance/StkTruToken.sol

```diff
@@ -176,6 +187,15 @@ contract StkTruToken is VoteToken, StkClaimableContract, IPauseableContract, Ree
         initalized = true;
     }
 
+    /**
+     * @dev Set tfUSDC address
+     * @param _feeToken Address of tfUSDC to be set
+     */
+    function setFeeToken(IERC20 _feeToken) external onlyOwner {
```

Comment:
It might be unsafe to change feeToken after some fees were paid. Maybe check if totalFarmRewards for the token is 0?